janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -245,6 +245,10 @@
                 "this parameter to true but we do recommend to reindex your indexes and don't use this parameter.",
             ConfigOption.Type.MASKABLE, false);
 
+    public static final ConfigOption<String> RETRY_ON_CONFLICT =
```

Comment:
I would use `Integer` here also because no other value is possible.

Also, not necessary but I think maybe it is better to have no default value here? I.e. use ES default value?
I would use `null` as a default value here and then would use `retry_on_conflict` in the `RestElasticSearchClient` only if it isn't null. For example:
```
if (retryOnConflict != null && request.getRequestType() == ElasticSearchMutation.RequestType.UPDATE) {
  requestData.putAll(ImmutableMap.of(retryOnConflictKey, retryOnConflict));
}
```

Reply:
cannot keep null as default value, as we have null check in ConfigOption.java, hence lets use 0 as default value. any other suggestions are welcome.